SignalMessaging/Views/ContactsViewHelper.m

```diff
@@ -284,7 +286,7 @@ - (void)presentMissingContactAccessAlertControllerFromViewController:(UIViewCont
                                                                   @"Button text which opens the settings app")
                                                         style:UIAlertActionStyleDefault
                                                       handler:^(UIAlertAction *_Nonnull action) {
-                                                          [[UIApplication sharedApplication] openSystemSettings];
+                                                          [CurrentAppContext() openSystemSettings];
```

Comment:
Should we change the button label / alert options if not in main app?

Reply:
Added a card - we're calling openSystemSettings in potentially lots of places and should probably handle it uniformly.